mathesar/rpc/explorations.py

```diff
@@ -6,6 +6,8 @@
 from modernrpc.core import rpc_method, REQUEST_KEY
 from modernrpc.auth.basic import http_basic_auth_login_required
 
+from db.connection import exec_msar_func
+
```

Comment:
We never import `exec_msar_func` in the endpoint definition files. Please create appropriate function under the `db/` namespace and use them here.